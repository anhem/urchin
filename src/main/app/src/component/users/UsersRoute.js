import React, {Component} from 'react'
import {Route, Switch} from 'react-router-dom'
import UsersContainer from './UsersContainer';
import EditUserContainer from './user/EditUserContainer';
import NewUserContainer from './user/NewUserContainer';

class UsersRoute extends Component {

    render() {
        return (
            <Switch>
                <Route exact path="/users" component={UsersContainer}/>
                <Route path="/users/new" component={NewUserContainer}/>
                <Route path="/users/:id" component={EditUserContainer}/>
            </Switch>
        )
    }
}

export default UsersRoute