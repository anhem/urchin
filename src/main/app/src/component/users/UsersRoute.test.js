import React from 'react';
import {shallow} from 'enzyme'
import toJson from 'enzyme-to-json';
import UsersRoute from "./UsersRoute";

describe('UsersRoute', () => {

    it('match snapshot', () => {
        expect(toJson(shallow(<UsersRoute/>))).toMatchSnapshot();
    });
});
