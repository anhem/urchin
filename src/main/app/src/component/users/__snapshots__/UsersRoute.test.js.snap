// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersRoute match snapshot 1`] = `
<Switch>
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(UsersContainer)",
        "type": [Function],
      }
    }
    exact={true}
    path="/users"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(NewUserContainer)",
        "type": [Function],
      }
    }
    path="/users/new"
  />
  <Route
    component={
      Object {
        "$$typeof": Symbol(react.memo),
        "WrappedComponent": [Function],
        "compare": null,
        "displayName": "Connect(EditUserContainer)",
        "type": [Function],
      }
    }
    path="/users/:id"
  />
</Switch>
`;
